man/org.freedesktop.systemd1.xml

```diff
@@ -2449,7 +2449,7 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly s ManagedOOMMemoryPressure = '...';
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
-      readonly s ManagedOOMMemoryPressureLimitPercent = '...';
+      readonly u ManagedOOMMemoryPressureLimitPermyriad = ...;
```

Comment:
hmm, so this an api break of course, but i guess we can get away with it given that we declared oomd unstable so far.

So my understanding now is that the dbus prop is called xyzPermyriad and the externally configurable option without the Permyriad suffix?

(never heard the word before i must admit)

Reply:
Yea, xyzPermyriad over dbus, xyz externally.

Permyriad is rarely used like permille (which is also in the code base). But they're real words so I figured it would be less ambiguous to have it named that way so the property doesn't have to be passed as a string with the unit of measure.